src/libServer/LookupServer.cpp

```diff
@@ -40,9 +43,59 @@ using namespace std;
 CircularArray<std::string> LookupServer::m_RecentTransactions;
 std::mutex LookupServer::m_mutexRecentTxns;
 
+namespace {
 const unsigned int PAGE_SIZE = 10;
 const unsigned int NUM_PAGES_CACHE = 2;
 const unsigned int TXN_PAGE_SIZE = 100;
+const unsigned UINT8_ADDR_SIZE = ACC_ADDR_SIZE * 2;
+
+bool HasZilHrp(const string& input) {
```

Comment:
I think this could actually be reduced to just
```
inline bool HasZilHrp(const string& input) {
   return input.substr(0, 4) == string("zil1");
}
```
If above is used, then the boost hpps can also be un-included

Reply:
the boost string_view is actually a read-only wrapper to allow substring without constructing a new string.
But I guess since this is a short string, it should be ok. Will reduce this. :)